new/resource-map-locking.md

```diff
@@ -0,0 +1,67 @@
+---
+RFC: unassigned
+Title: Resource Map Locking
+Author: Noah Kantrowitz <noah@coderanger.net>
+Status: Draft
+Type: Standards Track
+---
+
+# Resource Map Locking
+
+To improve the migration process of resources from cookbooks to core, as well
+as prevent user confusion, resources in cookbooks will no longer be allowed to
```

Comment:
This is already true today - core wins and that's what's caused the problems, I thought?

Perhaps this should say "resources migrated from cookbooks will not override the cookbook until we want it to (on Chef major version boundaries)" ?

Reply:
Core wins because we put lines like this into any resource in order to try to make core more specific:

```ruby
provides(:dmg_package) { true }
```

If someone had the same thing in a cookbook it would win.